lib/evm_database_ops.rb

```diff
@@ -110,6 +111,7 @@ def self.restore(db_opts, connect_opts = {})
     if db_opts[:local_file].nil?
       if action == :restore
         uri = connect_opts[:uri]
+        connect_opts[:remote_file_name] ||= connect_opts[:uri]
```

Comment:
I think we can remove this and use `uri` (`remote_file_uri`) in the block, right?

Reply:
Yes!  I knew there was something we changed that could handle that.  Thanks.  Changing now.